webpack.config.dev.js

```diff
@@ -36,7 +34,19 @@ export default {
         collapseWhitespace: true
       },
       inject: true
-    })
+    }),
+    new webpack.LoaderOptionsPlugin({
+      minimize: false,
+      debug: true,
+      noInfo: true, // set to false to see a list of every file being bundled.
+      options: {
+        sassLoader: {
+          includePaths: [path.resolve(__dirname, 'src', 'scss')]
+        },
+        context: '/',
+        postcss: () => [autoprefixer],
+      }
+    }) 
   ],
   module: {
     loaders: [
```

Comment:
Per [docs](https://webpack.js.org/guides/migrating/) `module.loaders` is now `module.rules`